### PR TITLE
update boilerplate to reflect framework datatype

### PR DESCRIPTION
**Pull request by HugoPeters1024**
relates to https://github.com/digitallyinduced/ihp/pull/485

### CODE REVIEW

File: Config/Config.hs [hunk 1]
```diff
@@ -4,6 +4,10 @@ import IHP.Prelude
 import IHP.Environment
 import IHP.FrameworkConfig
 
-instance FrameworkConfig where 
+
+config :: IO FrameworkConfig
+config = let
```
Comment (mpscholten) [inline Config/Config.hs:9]: This way it's not very clear how to e.g. override attributes

E.g. having it this way it's more clear:


```haskell
config = do
    let environment = Development
    let appHostName = "localhost"

    config <- defaultFrameworkConfig environment appHostName

    config
        |> set #mailServer SES { accessKey = "..", secretKey = "..", region = "eu-west-1" }
        |> modify #sessionCookie (\cookie -> cookie {
            setCookieMaxAge = Just (fromIntegral (60 * 60 * 24 * 90))
        })

Comment (HugoPeters1024) [reply Config/Config.hs:9]: Agreed! cool syntax!

Comment (HugoPeters1024) [reply Config/Config.hs]: Expanding defaultFrameworkConfig into developer land to solve the late binding issue also fixed this

**Comment by mpscholten** [comment]
Just saw this PR is still open, as this is live now I'm closing this 🚀

**Pull request closed without merging**